bedRModv2.tex

```diff
@@ -0,0 +1,393 @@
+        4 
+        & \textsf{name} 
+        & String 
+        & \texttt{[{\textbackslash}x20-{\textbackslash}x7e]\{1,255\}} 
+        & Modification name and additional attributes \\
+        5 & \textsf{score} & String & \texttt{[{\textbackslash}x20-{\textbackslash}x7e]\{1,255\}} & Modification confidence \\
+        6 & \textsf{strand} & String & \texttt{[-+.]} & \textbf{Feature} strand \\
+        7 & \textsf{thickStart} & Int & $[0, 2^{64}-1]$ & Thick start position, typically same as~\textsf{chromStart} \\
+        8 & \textsf{thickEnd} & Int & $[0, 2^{64}-1]$ & Thick end position, typically same as~\textsf{chromEnd} \\
+        9 & \textsf{itemRgb} & Int,Int,Int & \texttt{(}$[0, 255], [0,255], [0,255]$\texttt{) | 0} & Display color \\ % chktex 9
+        10 & \textsf{coverage} & Int &  $(0, 2^{64}-1]$ & Coverage \\
+        11 & \textsf{frequency} & Float & $[0, 100]$ & Percentage of modification \\
+        \bottomrule
+      \end{tabularx}
+    \end{adjustwidth}
+    \caption{\textbf{\acs{bedRMod} Fields.}}\label{tab:fields}
+  \end{table}
+\end{savenotes}
+
+In a \ac{bedRMod}~\textbf{file}, each~\textbf{data line} must have the same number of~\textbf{field}s.
+The positions in~\textbf{\acs{bedRMod} field}s are all described in the~\textbf{0-based, half-open coordinate system}, exactly as 
+described in the official \ac{BED} specifications.
+
+\subsection{Coordinates}
+Refer to the official \ac{BED} specifications.
+
+\subsection{Simple attributes}
+\begin{enumerate}
+\item \textsf{name}: String that describes the~\textbf{feature}, \textit{i.e.} the modification. \textsf{name} must describe 
+the modification using the \emph{short name} of the MODOMICS nomenclature\footnote{MODOMICS, \url{https://www.genesilico.pl/modomics/modifications}}, or the base modification code described in the SAMtags\footnote{SAM tags, \url{https://samtools.github.io/hts-specs}}, or a numeric ChEBI code\footnote{Chemical Entities of Biological Interest, \url{https://www.ebi.ac.uk/chebi}}. The MODOMICS \emph{short name} corresponding to~\textsf{name} is always described in the~\textbf{header field}~\textsf{modification\textunderscore names}. Additional name attributes are allowed, and must be comma-separated \textit{e.g.}~\texttt{a,DRACH,2}.
+
+\item \textsf{score}: String representation of the confidence in calling this modification. Any measure of confidence is valid, but a \ac{bedRMod}~\textbf{file} with non-integer-like~\textsf{score} values outside the range $[0, 1000]$ may fail from being correctly displayed in a visual representation\footnote{\textit{cf.} bedtools definition of score, \url{https://bedtools.readthedocs.io/en/latest/content/general-usage.html?highlight=bed\%20format}}.
+
+\item \textsf{coverage}: Integer between~0 and the maximum size of an unsigned 64-bit integer, excluding~0, representing the number of reads aligned at this position or the number of calls for this~\textbf{feature}.
```

Comment:
👍

Reply:
Exact definition of `coverage` might need to be adjusted, see above.